### PR TITLE
SW-4888 Feature flag for deliverables MVP

### PR DESCRIPTION
**Pull request by karthikbtf**
- Nick, do we want to reuse this for the Accelerator Console as well or have 2 flags?

### CODE REVIEW

File: src/features.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import env from 'src/utils/useEnvironment';
 import { CachedUserService } from 'src/services';
 
-export type FeatureName = 'Show Production View' | 'Projects' | 'User Detailed Sites';
+export type FeatureName = 'Show Production View' | 'Projects' | 'User Detailed Sites' | 'Deliverables MVP';
```
Comment (karthikbtf) [inline src/features.ts:4]: I can't seem to list these on separate lines, the formatter brings them back to the same line.

**Comment by nickgraz** [comment]
I thought we'd decided to just do one feature flag for the whole "accelerator in terraware" stuff?

**Comment by karthikbtf** [comment]
> I thought we'd decided to just do one feature flag for the whole "accelerator in terraware" stuff?

Was wondering if we wanted to use this flag as the accelerator flag - since this entire effort goes out as one.
I can separate the flags, might be less confusing.

**Comment by nickgraz** [comment]
I was thinking you can just rename this flag to "Accelerator" or something and we can use it across the board.

**Comment by karthikbtf** [comment]
> I was thinking you can just rename this flag to "Accelerator" or something and we can use it across the board.

Cool, will do.

**Review by nickgraz — APPROVED** [review]

**Pull request merged**